testing/upload_validation.ts

```diff
@@ -1268,13 +1275,16 @@ const packMetadataSchemaBySdkVersion: SchemaExtension[] = [
 
         // Pack has multiple network domains and user auth. The code needs to clarify which domain gets the auth
         // headers.
-        if (!data.networkDomains?.includes(data.defaultAuthentication.networkDomain)) {
-          context.addIssue({
-            code: z.ZodIssueCode.custom,
-            path: ['defaultAuthentication.networkDomain'],
-            message: 'The `networkDomain` in setUserAuthentication() must match a previously declared network domain.',
-          });
-          return;
+        for (const authNetworkDomain of authNetworkDomains) {
+          if (!data.networkDomains?.includes(authNetworkDomain)) {
```

Comment:
can you also validate that the network domain has no space characters? I just saw that you use that to encode the list.

Reply:
Done

Reply:
> Mind updating the CHANGELOG to include a new [Unreleased] entry with this?

Done